cypress/e2e/supervision/clients/documents/open-document.cy.js

```diff
@@ -0,0 +1,68 @@
+import path from "path";
+
+beforeEach(() => {
+  cy.loginAs("Allocations User");
+  cy.createAClient();
+  cy.loginAs("Public API");
+  cy.uploadDocument();
+});
+
+describe(
+  "Open document successfully",
+  { tags: ["@supervision-core", "@documents", "@open-document", "@smoke-journey"] },
+  () => {
+    it("opens a document successfully", () => {
+      cy.loginAs("Case Manager");
+      cy.get("@clientId").then((clientId) => {
+        cy.visit(`/supervision/#/clients/${clientId}`);
+        cy.contains("Ted Tedson");
+      });
+
+      cy.get(".TABS_DOCUMENTS").click();
+      cy.get(".filter-numbers > .number").should("have.text", 1);
+      cy.get("#select-all-documents-checkbox").check({ force: true });
+
+      cy.task('listContentsOfDownloadsFolder', Cypress.config("downloadsFolder")).then(beforeDownloadList => {
+        cy.contains(".button", "Open").click().wait(1000);
+        cy.task('listContentsOfDownloadsFolder', Cypress.config("downloadsFolder")).then(afterDownloadList => {
+          const newFilename = afterDownloadList.filter(file => !beforeDownloadList.includes(file))[0]
+          const newFilePath = path.join(
+            Cypress.config("downloadsFolder"),
+            newFilename
+          );
+          cy.verifyDownload(newFilename, {contains: true});
+          cy.readFile(newFilePath)
```

Comment:
You might not need the readFile given you have verify download. Also I seem to remember you need to chain a `.should('exist')` if you want it to assert the file is there.

Reply:
Docs now suggest you should just use `cy.readFile('xxx')` - https://docs.cypress.io/api/commands/readfile

Reply:
Ah, awesome. Should have checked the docs first!